File: Clase1/Lab02/20222828_Lab_2 (1)/20222828_Lab_2/Assessment.java
import java.util.Date;
import java.util.List;


//Cuando me habla de clases que representan una relación de composicion
// Relacion de 1 a muchos, ejm: casa 1 --- 1, * habitaciones

class Assessment {
	private int durationMinutes;
	private Date startDate;
	private List<Teacher> teachers;
	private List<AssessmentItem> items;

	public Assessment(int durationMinutes, Date startDate, List<Teacher> teachers, List<AssessmentItem> items){
		this.durationMinutes = durationMinutes;
		this.startDate = startDate;
		this.teachers = teachers;
		this.items = items;
	}

	public int getDurationMinutes() {
		return durationMinutes;
	}

	public Date getStartDate() {
		return startDate;
	}

	public List<Teacher> getTeachers() {
		return teachers;
	}

	public List<AssessmentItem> getItems() {
		return items;
	}
}

File: Clase6_Lab4/Laboratorio4-Fuentes-C# (1)/Laboratorio4-Fuentes-C#/AssessmentConsoleApp/Assessment.cs
using System;
using System.Collections.Generic;

public class Assessment
{
    private int durationInMinutes;
    private DateTime startDate;
    private List<Teacher> teachers;
    private List<AssessmentItem> assessmentItems;
    private double finalScore;

    public Assessment(int durationInMinutes, DateTime startDate, List<Teacher> teachers, List<AssessmentItem> assessmentItems)
    {
        this.durationInMinutes = durationInMinutes;
        this.startDate = startDate;
        this.teachers = teachers;
        this.assessmentItems = assessmentItems;
    }

    public int DurationInMinutes
    {
        get { return durationInMinutes; }
    }

    public DateTime StartDate
    {
        get { return startDate; }
    }

    public List<Teacher> Teachers
    {
        get { return teachers; }
    }

    public List<AssessmentItem> AssessmentItems
    {
        get { return assessmentItems; }
    }
}